### PR TITLE
Add `CONTRIBUTING.md`

### PR DESCRIPTION
**Pull request by Nadrieril**
This adds a first attempt at contribution instructions.

### CODE REVIEW

File: CONTRIBUTING.md
Comment (Nadrieril) [reply CONTRIBUTING.md:19]: This is commented out, we don't have documentation yet

File: CONTRIBUTING.md
Comment (Nadrieril) [reply CONTRIBUTING.md:100]: The project maintainers. We don't want random contributors to go chat with Eurydice maintainers, do we?

Comment (Nadrieril) [reply CONTRIBUTING.md]: I added a precision

File: CONTRIBUTING.md
Comment (sonmarcho) [inline CONTRIBUTING.md:45]: We should put this in an issue template

**Review by sonmarcho — APPROVED** [review]
Looks good to me!
But I suggest waiting for the other reviews

**Comment by R1kM** [comment]
Looks good to me too, at least as an initial template (we can always extend later). Feel free to merge.

**Comment by sonmarcho** [comment]
I also added @msprotz as a reviewer

**Review by msprotz — APPROVED** [review]
👍

**Comment by sonmarcho** [comment]
The branch is good to merge.

File: CONTRIBUTING.md
Comment (Nadrieril) [reply CONTRIBUTING.md:45]: Tracker in https://github.com/AeneasVerif/charon/issues/198

**Pull request merged**
